pkg/volume/rbd/attacher.go

```diff
@@ -213,13 +233,31 @@ func (detacher *rbdDetacher) UnmountDevice(deviceMountPath string) error {
 		return err
 	}
 	klog.V(3).Infof("rbd: successfully umount device mountpath %s", deviceMountPath)
+	klog.V(4).Infof("rbd: detaching device %s", mp.Device)
 
-	klog.V(4).Infof("rbd: detaching device %s", devicePath)
-	err = detacher.manager.DetachDisk(detacher.plugin, deviceMountPath, devicePath)
+	// add some retries for detach operation, and wait some time between two tries
+	var maxRetryCount uint = 3
+	for i := uint(0); ; i++ {
```

Comment:
`wait.ExponentialBackoff`? It will do the loop + sleep with exponentially growing sleep interval.

Reply:
i am not familiar with this function before, now i replace the logic with this function. thank you.